modules/http-api/src/main/java/org/mule/runtime/http/api/ws/FragmentHandler.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.ws;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Allows to channel the information of a multi fragment Web Socket message into an {@link InputStream}.
+ * <p>
+ * Clients <b>MUST</b> always invoke either {@link #complete()} or {@link #abort()} once the multi fragment
+ * message is over.
+ *
+ * @since 4.1.5 as experimental
+ */
+@Experimental
+@NoImplement
+public interface FragmentHandler {
```

Comment:
Why is this part of the API? I don't see any references so I can't understand why you would need it at this level.

Reply:
because I suck. moved to EE